java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -56,6 +58,91 @@ public interface ListBaseCommands {
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/lpos/">redis.io</a> for details.
+     * @param key The name of the list.
+     * @param element The value to search for within the list.
+     * @param options The LPos options.
+     * @return The index of <code>element</code>.
+     * @example
+     *     <pre>{@code
+     * assertEquals(6L, client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e"}).get());
+     * // Returns the second occurrence of the element "e".
+     * LPosOptions options = LPosOptions.builder().rank(2L).build();
+     * assertEquals(5L, client.lpos("my_list", "e", options);
+     *
+     * // rank and maxLength
+     * LPosOptions options = LPosOptions.builder().rank(1L).maxLength(1000L).build();
+     * assertEquals(4L, client.lpos("my_list", "e", options);
+     * }</pre>
+     */
+    CompletableFuture<Long> lpos(
+            @NonNull String key, @NonNull String element, @NonNull LPosOptions options);
+
+    /**
+     * Returns an array of indices of matching elements within a list. If no match is found, <code>
+     * null</code> is returned.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/lpos/">redis.io</a> for details.
+     * @param key The name of the list.
+     * @param element The value to search for within the list.
+     * @param count The number of matches wanted.
+     * @return An array that holds the indices of the matching elements within the list.
+     * @example
+     *     <pre>{@code
+     * assertEquals(new Long[]{7L}, client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e", "e"}).get());
+     * assertEquals(new Long[]{4L, 5L, 6L}, client.lposCount("my_list", "e", 3L).get());
+     * }</pre>
+     */
+    CompletableFuture<Long[]> lposCount(@NonNull String key, @NonNull String element, long count);
+
+    /**
+     * Returns an array of indices of matching elements inside a list. If no match is found, <code>
+     * null</code> is returned.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/lpos/">redis.io</a> for details.
+     * @param key The name of the list.
+     * @param element The value to search for within the list.
+     * @param count The number of matches wanted.
+     * @param options The LPos options.
+     * @return An array that holds the indices of the matching elements within the list.
+     * @example
+     *     <pre>{@code
+     * assertEquals(6L, client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e", "e"}).get());
+     *
+     * // rank
+     * LPosOptions options = LPosOptions.builder().rank(2L).build();
+     * assertEquals(new Long[]{5L}, client.lposCount("my_list", "e", 1L, options).get());
+     *
+     * // rank and maxLength
+     * LPosOptions options = LPosOptions.builder.rank(2L).maxLength(1000L).build();
+     * assertEquals(new Long[]{5L, 6L}, client.lposCount("my_list", "e", 3L, options).get());
```

Comment:
Same as above, not sure if this is necessary or not
```suggestion
     * LPosOptions options1 = LPosOptions.builder().rank(2L).build();
     * assertEquals(new Long[]{5L}, client.lposCount("my_list", "e", 1L, options1).get());
     *
     * // rank and maxLength
     * LPosOptions options2 = LPosOptions.builder.rank(2L).maxLength(1000L).build();
     * assertEquals(new Long[]{5L, 6L}, client.lposCount("my_list", "e", 3L, options2).get());
```